docs/tables/aws_s3_server_access_log/queries.md

```diff
@@ -0,0 +1,272 @@
+## Access Log Examples
+
+### Daily access trends
+
+Count access log entries per day to identify trends over time.
+
+```sql
+select
+  strftime(timestamp, '%y-%m-%d') AS access_date,
+  count(*) AS access_count
+from
+  aws_s3_server_access_log
+group by
+  access_date
+order by
+  access_date asc;
+```
+
+### Top 10 accessed objects
```

Comment:
This query should return a list of 10 objects, what bucket they're in, and how many times they were accessed. We should also avoid looking where the key name is `nil`